.github/workflows/performance.yml

```diff
@@ -30,6 +30,11 @@ concurrency:
 permissions: {}
 
 env:
+  # Performance testing should be performed in an environment reflecting a standard production environment.
+  WP_DEBUG: false
+  SCRIPT_DEBUG: false
```

Comment:
@joemcgill Per [Benchmarking PHP performance with Server-Timing](https://make.wordpress.org/performance/handbook/measuring-performance/benchmarking-php-performance-with-server-timing/#preparing-a-wordpress-site-for-server-timing-benchmarks) document. 

> (Optional, but recommended) Disable any debugging features of WordPress, e.g. set constants like WP_DEBUG, SCRIPT_DEBUG, SAVEQUERIES, to false.

Could we add `SAVEQUERIES` to false also?